Stores/src/standard/java/com/infomaniak/lib/stores/ui/dialogs/ReviewAlertDialog.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Infomaniak kDrive - Android
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.lib.stores.ui.dialogs
+
+import android.content.Context
+import android.view.LayoutInflater
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import com.infomaniak.lib.stores.databinding.AlertDialogReviewBinding
+
+class ReviewAlertDialog(
+    private val activityContext: Context,
+    private val customThemeRes: Int?,
+    private val reviewAlertDialogData: ReviewAlertDialogData,
+) {
+
+    private val viewBinding = AlertDialogReviewBinding.inflate(LayoutInflater.from(activityContext))
+
+    fun show() {
+        val builder = customThemeRes?.let {
```

Comment:
I just thought about placing the dialog creatiion in an 'init {}', that would feel more natural at the usage, where all the parameters are supply to the constructor and not to the show() function